src/esp/sim/Simulator.h

```diff
@@ -308,6 +311,8 @@ class Simulator {
 
   gfx::WindowlessContext::uptr context_ = nullptr;
   std::shared_ptr<gfx::Renderer> renderer_ = nullptr;
+  gfx::ShaderManager shaderManager_;
```

Comment:
Stored at this level, we will likely have to pass this down into the `loadScene()` pipeline also to create the correct default shaders for the scene `DrawableGroups` based on `AssetInfo` types. It will also change the C++ test setup pipeline at that point.

Reply:
For now, I was planning on passing this into the ResourceManager for use when drawables are created.